### DISCUSSION TITLE
DNS configured and propagated correctly but site not loading

### DISCUSSION THREAD

**Discussion by Iamstanlee** [vercel/community#3148] [Help]
### Summary

Hey, 
I have my DNS configured and setup on vercel, Everything looks good and propagating correctly as I checked using [DNSChecker](https://dnschecker.org/#A/stanlee.dev)

For some reason the site is not loading and keeps timing out with `ERR_TIMED_OUT`, It was working fine as at yesterday

Affected domain: stanlee.dev

### Steps to Reproduce

 - Check DNS propagation using `dig @8.8.8.8 +short A stanlee.dev`or [DNSChecker](https://dnschecker.org/#A/stanlee.dev)
 - Affected domain stanlee.dev

**Comment by amyegan** [comment]
Hi @Iamstanlee! The site at stanlee.dev is loading for me now. Are you still seeing this problem?

**Reply by Iamstanlee** [reply]
Hey @amyegan, thanks for the reply, I had to move to another provider as it was a bit urgent at that point
